worldedit-core/src/main/java/com/sk89q/worldedit/function/operation/Operation.java

```diff
@@ -56,7 +63,29 @@
      * of the operation.
      *
      * @param messages The list to add messages to
+     * @deprecated Will be removed in WorldEdit 8.0 - use the Component variant
      */
-    void addStatusMessages(List<String> messages);
+    @Deprecated
+    default void addStatusMessages(List<String> messages) {
+    }
 
+    Set<String> warnedDeprecatedClasses = new HashSet<>();
```

Comment:
Should we add javadoc to mark this as internal? I know people aren't likely to use it, but it is currently public.